### DISCUSSION TITLE
Genuine Concern: Ravynos and Hardware Support

### DISCUSSION THREAD

**Discussion by st0rm-shad0w** [ravynsoft/ravynos#459] [General]
Since Ravynos is BSD-based, is there a real risk of limited hardware support, especially for network devices, or is this concern exaggerated?

**Comment by mszoek** [comment]
It's exaggerated for network devices. FreeBSD has excellent support for wired networking, which is why it gets used for a lot of high-scale servers. Wireless can be iffy for newer devices, but their strategy of porting from Linux is pretty solid and they support a lot of chipsets. The concern is more realistic for desktop devices, especially GPUs and sound. That said, it's not impossible and there's constant progress in the BSD community. People are also welcome to help create or port drivers for those missing parts.

**Reply by Pwrsrfrvr** [reply]
Curious,what are the minimum system requirements?

**Reply by mszoek** [reply]
For the FreeBSD-based versions up to 0.6.x, it should run on any UEFI x86_64 system with 2+ CPUs clocked around 1.5-2 GHz and 2GB RAM. USB keyboard/mouse work best. GPU is not accelerated so it will run in VMs but graphics are slow on lower end systems.

For the upcoming Darwin-based version 0.7 and beyond, probably similar but TBD :)
